src/components/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const initialState = [];
+
```

Comment:
- [ ] Same here, Please define the `actionTypes` for checking the status action to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).